fix(curriculum): changed challenge test text to use the word should for Responsive Web Design (#36860)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/comment-out-html.english.md

```yml
tests:

>   - text: Commenting out your <code>h1</code> element should <code>h1</code> element not visible on your page.

Reviewer: ```suggestion
  - text: Your <code>h1</code> element should be commented out so that it is not visible on the page.
```

    testString: assert(($("h1").length === 0));

>   - text: Leaving your <code>h2</code> element uncommented should mke the <code>h2</code> element visible on your page.

Reviewer: ```suggestion
  - text: Your <code>h2</code> element should not be commented out so that it is visible on the page.
```

    testString: assert(($("h2").length > 0));

>   - text: Commenting out your <code>p</code> element should <code>p</code> element not visible on your page.

Reviewer: ```suggestion
  - text: Your <code>p</code> element should be commented out so that it is not visible on the page.
```
